weekly.go

```diff
@@ -149,6 +149,14 @@ func runWeelyCommandFunc(cmd *cobra.Command, args []string) {
 	title := sprint.Name
 	createWeeklyReport(title, body.String())
 
+	pendingIssues := queryJiraIssues(
+		fmt.Sprintf("project = %s and Sprint = %d and statusCategory != Done",
+			config.Jira.Project, sprint.ID,
+		))
+	// Move issues then active the next sprint.
+	moveIssuesToSprint(nextSprint.ID, pendingIssues)
+	updateSprintState(nextSprint.ID, "active")
```

Comment:
if there is an active sprint, can we enable another one?

Reply:
There can be multiple active sprints if we enable Parallel Sprints features(but we do not enable it).